addons/web/static/tests/core/l10n/translation_tests.js

```diff
@@ -171,3 +171,21 @@ QUnit.test("tamil has the correct numbering system", async (assert) => {
         "௧௦/௧௨/௨௦௨௧ ௧௨:௦௦:௦௦"
     );
 });
+
+QUnit.test(
+    "_t fills the format specifiers in translated terms with its extra arguments",
+    async (assert) => {
+        const translatedStr = _t("Due in %s days", 513);
+        assert.strictEqual(translatedStr, "Due in 513 days");
+    }
+);
+
+QUnit.test(
+    "_t fills the format specifiers in lazy translated terms with its extra arguments",
+    async (assert) => {
+        translatedTerms[translationLoaded] = false;
+        const translatedStr = _t("Due in %s days", 513);
+        translatedTerms[translationLoaded] = true;
+        assert.equal(translatedStr, "Due in 513 days");
+    }
+);
```

Comment:
Please add a translation term for these to check that interpolation works in conjunction with translation

Reply:
changes done